app/views/support_tickets/thank_you.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :title,  'Thank you for completing this form' %>
+
+<%- content_for :before_content do %>
+  <% breadcrumbs([{ 'Home' => guidance_page_path },
+                  {t('landing_pages.get_support.start') => support_ticket_path},
+                  'Thank you for completing this form',
+                 ]) %>
+<% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">
+      Thank you for completing this form
+    </h1>
+    <p class="govuk-body">Our support team has received your request. If you do not receive an email from us within that time, please check your spam or junk folder.</p>
+
+    <p class="govuk-body">We aim to respond within 3 working days.</p>
```

Comment:
Seems like the sentences here are slightly out-of-order - shouldn't "If you do not receive an email from us within that time"  come _after_ "We aim to respond within 3 working days" ?